docs/docs/recipes/manage-users/README.md

```diff
@@ -4,8 +4,19 @@ sidebar_position: 6
 
 # 🧑‍🚀 Manage users
 
-TODO: briefly summary
+User management is the core feature for the admin staff.
```

Comment:
"admin staff", not so good.

Reply:
just "admin users"?

Reply:
you don't need to say admin here

Reply:
just

> User management is the core feature.

Reply:
After simplified, it seems that this sentence is unnecessary now.

Removed.